authors/apps/followers/models.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+from authors.settings import AUTH_USER_MODEL
+from authors.apps.authentication.models import User
+
+# Create your models here.
```

Comment:
You can remove this comment because that is definitely a model

Reply:
I have fixed this